manifests/v2/base/manager/manager.yaml

```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: training-operator-v2
+  labels:
+    training.kubeflow.org/component: manager
```

Comment:
Instead of these, could we use the kubernetes recommended labels?

https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/